internal/common/db.go

```diff
@@ -144,3 +151,18 @@ func GetHostFromDB(db *gorm.DB, clusterId, hostId string) (*Host, error) {
 func DeleteRecordsByClusterID(db *gorm.DB, clusterID strfmt.UUID, value interface{}, where ...interface{}) error {
 	return db.Where("cluster_id = ?", clusterID).Delete(value, where...).Error
 }
+
+func (c *Cluster) AfterFind(db *gorm.DB) error {
```

Comment:
Where is it used?

Reply:
this is a hook being invoked by GORM: https://gorm.io/docs/hooks.html#Object-Life-Cycle